packages/table/src/interactions/selectable.tsx

```diff
@@ -110,142 +124,92 @@ export class DragSelectable extends React.Component<IDragSelectableProps, {}> {
     }
 
     private handleActivate = (event: MouseEvent) => {
+        const { locateClick, selectedRegions, selectedRegionTransform } = this.props;
```

Comment:
moving to this approach for consistency with other table code (and for shorter line length in some cases)